Sources/WalletConnectUtils/SetStore.swift

```diff
@@ -0,0 +1,21 @@
+
+import Foundation
+
+public actor SetStore<T: Hashable> {
```

Comment:
I think more convenient to make it thread safe with .async(flags: .barrier) on write and .sync in read

Reply:
and we no need asyncFilter, and async Task wrapping

Reply:
you are totally right!